projects/demo/src/modules/directives/dropdown/index.ts

```diff
@@ -16,6 +17,7 @@ import {DropdownDocumentation} from '../../components/abstract/dropdown-document
         TuiActiveZone,
         TuiButton,
         TuiDemo,
```

Comment:
`TuiDocDropdown` should be inside `TuiDemo`

Reply:
Other components do not need it, but only the dropdowns examples. If we put it inside `TuiDemo`, it seems that this will be an extra import for those who do not need it?

Reply:
Oh, sorry